AppConnectSwift/SecurityQuestionViewController.swift

```diff
@@ -8,6 +8,10 @@ class SecurityQuestionViewController: UIViewController, UITableViewDelegate, UIT
     var createAccountViewController = CreateAccountViewController()
     var userEmail : String!
     var userPassword : String!
+    let securityQuestionKey = "user_security_questions"
+    let depricatedKey = "deprecated"
+    let idKey = "id"
+    let questionKey = "name"
```

Comment:
Nitpick: these should probably be static to be consistent with our usage elsewhere